data/json/monster_weakpoints/abomination_weakpoints.json

```
@@ -0,0 +1,339 @@
+          },
+          {
+            "effect": "maimed_leg",
+            "chance": 25,
+            "duration": 86400,
+            "message": "The %s's leg is disabled!",
+            "damage_required": [ 31, 100 ]
+          }
+        ],
+        "coverage": 2
+      },
+      {
+        "id": "head",
+        "name": "the head",
+        "damage_mult": { "all": 1.1 },
+        "crit_mult": { "all": 1.1 },
+        "armor_mult": { "physical": 0.75 },
+        "difficulty": { "melee": 3, "ranged": 5 },
+        "coverage": 3
+      },
+      {
+        "id": "eye",
+        "name": "the eye",
+        "armor_mult": { "physical": 0.4 },
+        "crit_mult": { "all": 1.25 },
+        "difficulty": { "ranged": 8, "melee": 6 },
+        "effects": [
+          {
+            "effect": "blind",
+            "duration": 86400,
+            "chance": 20,
+            "message": "The %s's eyes are obliterated!",
+            "damage_required": [ 35, 100 ]
+          }
+        ],
+        "coverage": 2
+      }
+    ]
+  },
+  {
+    "type": "weakpoint_set",
+    "id": "wps_amalgamation_acid",
+    "weakpoints": [
+      {
+        "id": "acid_gland",
+        "name": "the acid gland",
+        "armor_mult": { "physical": 0.5 },
+        "crit_mult": { "all": 1.25 },
+        "difficulty": { "ranged": 5, "melee": 5 },
+        "effects": [
+          {
+            "effect": "maimed_acid_gland",
+            "chance": 15,
+            "duration": [ 1, 10 ],
+            "message": "The %s's acid glands are maimed!",
+            "damage_required": [ 1, 15 ]
+          },
+          {
+            "effect": "maimed_acid_gland",
+            "chance": 33,
+            "duration": [ 5, 25 ],
+            "message": "The %s's acid glands are maimed!",
+            "damage_required": [ 16, 30 ]
+          },
+          {
+            "effect": "maimed_acid_gland",
+            "chance": 50,
+            "duration": 86400,
+            "message": "The %s's acid glands are destroyed!",
+            "damage_required": [ 31, 100 ]
+          }
+        ],
+        "coverage": 7
+      },
+      {
+        "id": "proboscis",
+        "name": "the proboscis",
+        "damage_mult": { "all": 0.5 },
+        "crit_mult": { "all": 0.5 },
+        "difficulty": { "ranged": 5, "melee": 3 },
+        "effects": [
+          {
+            "effect": "maimed_mandible",
+            "chance": 25,
+            "duration": [ 1, 10 ],
+            "message": "The %s's proboscis is maimed!",
+            "damage_required": [ 1, 15 ]
+          },
+          {
+            "effect": "maimed_mandible",
+            "chance": 50,
+            "duration": [ 5, 25 ],
+            "message": "The %s's proboscis is maimed!",
+            "damage_required": [ 16, 30 ]
+          },
+          {
+            "effect": "maimed_mandible",
+            "chance": 50,
+            "duration": 86400,
+            "message": "The %s's proboscis is maimed!",
```

```suggestion
            "message": "The %s's proboscis is destroyed!",
```
To match the other 1d mandible disabling message